pkg/app/piped/cloudprovider/cloudrun/cloudrun.go

```diff
@@ -38,6 +38,7 @@ var (
 
 type (
 	Service  run.Service
+	Services []*Service
```

Comment:
I think we don't need this alias, directly mention `[]*Serivce` type is better to me, wdyt.

Reply:
There are some benefits to doing so.
For example, we can reuse a Service method when we want to use something method in Services.
```
func (s *Service) HasFields(fields ...string) bool {}

func (s Services) FilterByFields(fields ...string) Services{
	ret := make(Services, 0, len(s))
	for i := range s {
		if s[i].HasFields(fields...) {
			ret = append(ret, s[i])
		}
	}
	return ret
}
```

Reply:
Get your point on using collection methods, but until now there are not thing like that which we need to use thing like `services.HasFields(fields...)`. And even when it comes, one simple for loop maybe a better choice for it simplicity. For the current source, `Service` and `Services` will be mixed in and hard to track while reading.

Reply:
That's make sense! I'll fix it